### PR TITLE
fixed conflicts and made sure keymaps compiled, and some newer (older) keymaps snuck in

### PR DESCRIPTION
**Pull request by Rossman360**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/keebio/viterbi/keymaps/met/keymap.c [hunk 1]
```diff
@@ -0,0 +1,323 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
```
Comment (noroadsleft) [inline keyboards/keebio/viterbi/keymaps/met/keymap.c]: You don't need this line. QMK does this automatically.

```suggestion
```

File: keyboards/keebio/viterbi/keymaps/met/keymap.c [hunk 2]
```diff
@@ -0,0 +1,319 @@
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
```
Comment (noroadsleft) [inline keyboards/keebio/viterbi/keymaps/met/keymap.c]: I meant only Line 3 before. *This* line *is* required:

```suggestion
#include QMK_KEYBOARD_H

// Each layer gets a name for readability, which is then used in the keymap matrix below.
```

Comment (Rossman360) [reply keyboards/keebio/viterbi/keymaps/met/keymap.c]: whoopsie

File: users/rossman360/rossman360.c [hunk 3]
```diff
@@ -1,3 +1,5 @@
+
```
Comment (fauxpark) [inline users/rossman360/rossman360.c]:
```suggestion
```

File: users/rossman360/rossman360.c [hunk 4]
```diff
@@ -1,3 +1,5 @@
...
+
```
Comment (fauxpark) [inline users/rossman360/rossman360.c]:
```suggestion
```

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**